### ISSUE TITLE
godot client crash when networking is introduced and hit play

### ISSUE THREAD

**Issue by Migar4** [godotengine/godot#37544]
**Godot version:**
3.2 stable
**OS/device including version:**
win 10
**Issue description:**

I tried adding some networking functionality to my game then when I tried it out the game is playable but the client always crash. Before adding networking it was not crashing, I got a backlog from the console which told me to post the issue here.

```text
CrashHandlerException: Program crashed
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[0] String::humanize_size (c:\users\nat\onedrive\desktop\godot-3.2\core\ustring.cpp:3332)
[1] String::humanize_size (c:\users\nat\onedrive\desktop\godot-3.2\core\ustring.cpp:3332)
[2] ScriptEditorDebugger::_parse_message (c:\users\nat\onedrive\desktop\godot-3.2\editor\script_editor_debugger.cpp:814)
[3] ScriptEditorDebugger::_notification (c:\users\nat\onedrive\desktop\godot-3.2\editor\script_editor_debugger.cpp:1382)
[4] ScriptEditorDebugger::_notificationv (c:\users\nat\onedrive\desktop\godot-3.2\editor\script_editor_debugger.h:60) [5] Object::notification (c:\users\nat\onedrive\desktop\godot-3.2\core\object.cpp:933)
[5] Object::notification (c:\users\nat\onedrive\desktop\godot-3.2\core\object.cpp:933)
[6] SceneTree::_notify_group_pause (c:\users\nat\onedrive\desktop\godot-3.2\scene\main\scene_tree.cpp:987)
[7] SceneTree::idle (c:\users\nat\onedrive\desktop\godot-3.2\scene\main\scene_tree.cpp:525)
[8] Main::iteration (c:\users\nat\onedrive\desktop\godot-3.2\main\main.cpp:2028)
[9] OS_Windows::run (c:\users\nat\onedrive\desktop\godot-3.2\platform\windows\os_windows.cpp:3152)
[10] widechar_main (c:\users\nat\onedrive\desktop\godot-3.2\platform\windows\godot_windows.cpp:162)
[11] _main (c:\users\nat\onedrive\desktop\godot-3.2\platform\windows\godot_windows.cpp:184)
[12] main (c:\users\nat\onedrive\desktop\godot-3.2\platform\windows\godot_windows.cpp:196)
[13] __scrt_common_main_seh (f:\dd\vctools\crt\vcstartup\src\startup\exe_common.inl:253)
[14] BaseThreadInitThunk
-- END OF BACKTRACE --
```

**Steps to reproduce:**

not sure what I should type here but there are the all the new functions I tried for it to crash

in my lobby:
```
func _ready():
	get_tree().connect("network_peer_connected", self, "player_connected")


func _on_Host_pressed():
	var host = NetworkedMultiplayerENet.new()
	var res = host.create_server(5400, 2)
	
	if res != OK:
		print("error creating server")
		return
	
	$Host.hide()
	$Join.disabled = true
	
	get_tree().set_network_peer(host)

func _on_Join_pressed():
	var host = NetworkedMultiplayerENet.new()
	host.create_client("192.168.1.76", 5400)
	
	get_tree().set_network_peer(host)
	
	$Join.hide()
	$Host.disabled = true

func player_connected(id):
	Global.otherPlayer = id
	var game = preload("res://Scenes/Level.tscn").instance()
	get_tree().get_root().add_child(game)
	hide()
```

in my level:
```       var thisPlayer = preload("res://Scenes/character.tscn").instance()
	thisPlayer.set_name(str(get_tree().get_network_unique_id()))
	thisPlayer.set_network_master(get_tree().get_network_unique_id())
	add_child(thisPlayer)
	
	if Global.otherPlayer:
		var otherPlayer = preload("res://Scenes/character.tscn").instance()
		otherPlayer.set_name(str(Global.otherPlayer))
		otherPlayer.set_network_master(Global.otherPlayer)
		add_child(otherPlayer)
```

in the character:
```func _process(delta):
         if is_network_master():
                ...
                if Global.otherPlayer:
			rpc_unreliable("_on_recv_info", Global.info)```

and the slave func:
```slave func _on_recv_info(info : int):
        ...```

**Minimal reproduction project:**

**Comment by Anutrix** [comment]
Put some debug message after each line or every few lines. It might help you find the issue if you know the last thing that happened.

**Comment by Migar4** [comment]
it only crashes with the custom built client but otherwise it works fine and all I got from the debug console is the one that's there there was nothing more

anyway is this normal for godot to consume this much memory?

![image](https://user-images.githubusercontent.com/46416930/78347236-ac3b5780-75fc-11ea-8b5b-608b9c966183.png)

**Comment by mhilbrunner** [comment]
This is likely fixed in 4.0 by https://github.com/godotengine/godot/issues/35872.
Backporting this fix may be worthwhile.
